src/less/mixins/private/dialog-mixins.less

```diff
@@ -71,9 +71,6 @@
         align-self: center;
         flex: 1 1 auto;
         margin: 0;
-        overflow: hidden;
-        text-overflow: ellipsis;
-        white-space: nowrap;
```

Comment:
In old iterations we used `line-clamp` https://css-tricks.com/almanac/properties/l/line-clamp/ which we can use to limit it to 2-3 lines.

Reply:
I agree with clamping, but we had a long discussion about this offline (see thread) and it was decided to wrap indefinitely.